common/src/IO/AssimpParser.cpp

```diff
@@ -650,8 +660,8 @@ std::unique_ptr<Assets::EntityModel> AssimpParser::doInitializeModel(
       {loadTexturesForMaterial(*scene, mesh->mMaterialIndex, m_path, m_fs, logger)});
   }
 
-  // load the reference pose as the only frame for this model
-  return loadSceneFrame(*scene, *model, modelPath)
+  // the entity browser will want to see frame 0 most of the time, pre-emptively load it
```

Comment:
Nah, it's not necessary. I just noticed that clicking on the the entity browser will load all the models for the preview and figured it'd be good to preload the first frame instead of hitting the disk twice. I'll remove it.

Reply:
I can't imagine it makes much of a difference. I prioritized minimizing the time the editor is unresponsive during map load for the small price of another delay when you open the entity browser.